examples/experimental/i3s-17-and-debug/components/tile-validator.js

```diff
@@ -169,6 +175,23 @@ export default class TileValidator extends PureComponent {
     this.setState({triangleMessages});
   }
 
+  compareGeometryBoundingVolumeVsTileBoundingVolume(tile) {
```

Comment:
Why are we checking separately for null and falsy? This seems like a bad API.

Reply:
Refactoring here [Refactoring PR](https://github.com/visgl/loaders.gl/pull/1323)